inyoka_theme_default/static/style/inyoka/planet.less

```diff
@@ -10,16 +10,10 @@
 
 .planet {
   &-newestarticles {
-    .list-inline {
-      text-align:justifed;
-
-      li {
-        padding: 0;
-
-        &:not(:last-child):after {
-          color: @gray-light;
-          content: ' |';
```

Comment:
yeah,  `.list-inline` is basically a inline-block with padding. However, on mobile devices the separation via '| ' will get interesting to style – especially if the list fills multiple lines.